lib/fluent/plugin/out_influxdb.rb

```diff
@@ -55,6 +55,14 @@ def configure(conf)
                                               time_precision: @time_precision,
                                               use_ssl: @use_ssl,
                                               verify_ssl: @verify_ssl
+    
+    existing_databases = @influxdb.list_databases.map { |x| x['name'] }
+    if existing_databases.include? @dbname
+        $log.info 'Dababase ' + @dbname + ' exists'
```

Comment:
If we go with `info`, adding "exists on influxdb(#{host}:#{port})" is more better.